source/content/guides/decoupled/overview/04-site.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Pantheon Front-End Sites
+subtitle: Site Creation
+description: Create and configure a Front-End Site.
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan, backlineint, cobypear, hckia]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/overview/site
+anchorid: site
+reviewed: "2023-03-16"
+contenttype: [guide]
+innav: [false]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides configuration information for Front-End Sites.
+
+## Prerequisites
+
+* The **Decoupled** tab is enabled in your dashboard after onboarding.
+
+* Your site repository is in GitHub.
+
+* You have a domain to use for the Front-End site.
+
+* You will use Next.js or Gatsby as the frontend framework.
+
+    <Alert title="Note"  type="info" >
+
+    You can use any JavaScript-based runtime, or any static site generator that outputs HTML/CSS/JS, but Pantheon only fully supports Next.js and Gatsby.
+
+    </Alert>
+
+* (Optional). You are using the latest version of Drupal, or WordPress for the CMS.
+
+* (Optional). If using a pre-configured starter kit template, you will use one of the following combinations:
+
+    * [Drupal and Next.js](/guides/decoupled/drupal-nextjs-frontend-starters/)
+
+    * [WordPress and Next.js](/guides/decoupled/wp-nextjs-frontend-starters/)
+
+    * [WordPress and Gatsby](/guides/decoupled/wp-gatsby-frontend-starters/)
+
+    <Alert title="Note"  type="info" >
+
+    Other configurations are possible but not fully supported.
+
+    </Alert>
+
+### Compatibility
+
+You can create an application from scratch by importing a repository or you can use the Pantheon starter kit templates to develop a Front-End Site.
+
+You can also create a site without a CMS. This is done by utilizing the import repository feature during site creation.
+
+Supported Versions:
+
+|  Tooling   |      Version    |
+|  :---:     |       :---:     |
+|  Gatsby    |    4 or higher  |
+| Next.js    |   12 or higher  |
+|  Drupal    |   9 or higher   |
+| WordPress  |   5.9 or higher |
+
+## Before You Use Pantheon Starter Kits
+
+The following components are required to create and configure a new decoupled project using a Front-End Site starter kit:
+
+1. Install [Composer](https://getcomposer.org/download/) globally.
+    * Composer is required for the CMS backends.
+
+1. Install [Terminus](/terminus/install) version 3.0.0 or higher.
```

Comment:
@whitneymeredith using Terminus and also the following three items about tokens are optional. This is presented as required.

Reply:
Updated! Thanks!!